Source/Scene/MetadataEntity.js

```diff
@@ -227,6 +227,7 @@ MetadataEntity.getProperty = function (
 
   if (defined(classProperty)) {
     value = classProperty.normalize(value);
+    value = classProperty.unpackVectorTypes(value);
```

Comment:
In the future we may want to consider adding a `result` parameter to `getProperty` so it doesn't need to allocate the cartesian object. Looking at `Cesium3DTileBatchTable` it doesn't take a `result` parameter for `getProperty` either, maybe because the type could be anything without the new stricter type system?